zerver/models.py

```diff
@@ -883,6 +883,12 @@ class UserProfile(AbstractBaseUser, PermissionsMixin):
     notification_sound = models.CharField(max_length=20, default='zulip')  # type: str
     wildcard_mentions_notify = models.BooleanField(default=True)  # type: bool
 
+    # Topic notifications.
+    enable_topic_desktop_notifications = models.BooleanField(default=False)  # type: bool
+    enable_topic_email_notifications = models.BooleanField(default=False)  # type: bool
+    enable_topic_push_notifications = models.BooleanField(default=False)  # type: bool
+    enable_topic_audible_notifications = models.BooleanField(default=False)  # type: bool
```

Comment:
So, I realize this will be an annoying project to change, but I think "topic_` as the name is too brief, and should be E.g. `enable_followed_topic_desktop_notifications`.  

Also, do we want want the `wildcard_mentions_notify` option in this bundle bundle, as `enable_followed_topic_wildcard_mentions_notify`, so our table can be complete?

Reply:
Let me start a chat.zulip.org thread before we change this.

Reply:
https://chat.zulip.org/#narrow/stream/3-backend/topic/topics.20I.20follow.20naming/near/854541